test/fixtures/internal-modules/index.js

```diff
@@ -0,0 +1 @@
+module.exports = require('internal/freelist');
```

Comment:
Not sure what do you mean. `js2c` drops extensions so this is how we require internal modules. This particular test checks that if we try it in userland module and there is `node_modules/internal/freelist.js` everything works as before.